### PR TITLE
new lesson

### PR DESCRIPTION
**Pull request by firas16**

### CODE REVIEW

File: src/AppBundle/Controller/LessonController.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace AppBundle\Controller;
```
Comment (yassin8) [inline src/AppBundle/Controller/LessonController.php:2]: espace

File: src/AppBundle/Form/LessonType.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Firas
+ * Date: 25/12/2016
+ * Time: 21:02
+ */
```
Comment (yassin8) [inline src/AppBundle/Form/LessonType.php]: delete please

File: src/AppBundle/Form/LessonType.php [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+class LessonType extends AbstractType
+{
+    /**
+     * @param FormBuilderInterface $builder
+     * @param array $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+
```
Comment (yassin8) [inline src/AppBundle/Form/LessonType.php:19]: delete spaces

Comment (firas16) [reply src/AppBundle/Form/LessonType.php:19]: ok

**Pull request merged**